projects/postgresql/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER sfrost@snowman.net
+
+RUN apt-get update && apt-get install -y make libreadline-dev zlib1g-dev bison flex
+# autoconf automake libtool curl tcl zlib1g-dev
+
+RUN git clone https://github.com/postgres/postgres.git
```

Comment:
Is there a reason not to use the official repo?  Is there a way to make this work across the various versions of PG instead of just the current development branch?

Reply:
Changed to the official repo. I think ossfuzz is meant to fuzz head, not necessarily a released version.